evergreen.yml

```diff
@@ -100,7 +100,7 @@ functions:
           set -o verbose
 
           if [ "${is_patch}" = "true" ]; then
-            wget -q https://github.com/dbradf/pypi-version-check/releases/download/v0.2.0/pypi-version-check
+            wget -q https://github.com/dbradf/pypi-version-check/releases/download/v0.2.1/pypi-version-check
```

Comment:
Something seems to have changed with the pypi json response this tool makes, it is fixed in the new version.